.circleci/config.yml

```diff
@@ -84,6 +84,50 @@ jobs:
             source env/bin/activate
             make entry-point-test
 
+  package_test:
+    working_directory: ~/nlp_primitives
+    parameters:
+      image_tag:
+        type: string
+        default: "python:3.8"
+    executor:
+      name: python
+      image_tag: << parameters.image_tag >>
+    steps:
+      - checkout
+      - attach_workspace:
+          at: ~/nlp_primitives
+      - run :
+          name: "Install Requirements"
+          command: |
+            virtualenv env -q
+            source env/bin/activate
+            pip install -r requirements.txt
+            pip install -r complete-requirements.txt
+      - install_test_requirements
+      - run :
+          name: "Create Source Distribution"
+          command: |
+            virtualenv env -q
+            source env/bin/activate
```

Comment:
Is it necessary to create this virtualenv here?

Reply:
Oops, no! Doesn't do anything after the 1st time, I think, but I'll remove. Thanks